fubectl.source

```diff
@@ -187,6 +187,46 @@ kwns() {
     watch kubectl get pod -n "$ns"
 }
 
+# [kbash] create a pod with a bash
+kbash() {
+    kubectl run --generator=run-pod/v1 my-shell --rm -i --tty --image ubuntu -- bash
+}
+
+# [kssh] create root shell on a node
+kssh() {
```

Comment:
I do not really like the name, as it is no `ssh`. `knodesh` sounds good to me, it is a shell on the node.

Reply:
`konsole`? :trollface:

Reply:
`kerminal`

Reply:
i love both. cannot decide.

Reply:
changed to `konsole`